infrastructure/Taskfile.yml

```diff
@@ -1124,6 +1124,27 @@ tasks:
       preconditions:
       - *require_site
 
+    site:backup:restore:db:
+      desc: Restore the database backup for a site/environment to the same or another site/environment.
+      deps: [lagoon:cli:config]
+      vars:
+        PROJECT: "{{.PROJECT}}"
+        PROJECT_ENV: '{{.PROJECT_ENV | default "main" }}'
+        SOURCE_PROJECT: "{{.SOURCE_PROJECT | default .PROJECT }}"
+        SOURCE_ENV: "{{.SOURCE_ENV | default .PROJECT_ENV }}"
+        TARGET_PROJECT: "{{.TARGET_PROJECT | default .PROJECT}}"
+        TARGET_ENV: "{{.TARGET_ENV | default .PROJECT_ENV }}"
+        TARGET_NAMESPACE: "{{.TARGET_PROJECT}}-{{.TARGET_ENV}}"
+        LOCAL_BACKUP_DESTINATION: "/tmp/{{.SOURCE_PROJECT}}-mariadb-backup.tar.gz"
+        REMOTE_BACKUP_DESTINATION: "/tmp/{{.SOURCE_PROJECT}}-{{.SOURCE_ENV}}-mariadb-prebackuppod.mariadb.sql"
+      prompt:
+        - Restore latest database backup from {{.SOURCE_PROJECT}}-{{.SOURCE_ENV}} to {{.TARGET_PROJECT}}-{{.TARGET_ENV}}
+      cmds:
+        - LAGOON_PROJECT={{.SOURCE_PROJECT}} BACKUP_TYPE=mariadb BACKUP_DESTINATION={{.LOCAL_BACKUP_DESTINATION}} ./task/scripts/lagoon-get-backup.sh
+        - kubectl cp -n {{.TARGET_NAMESPACE}} $(tar -xvzf {{.LOCAL_BACKUP_DESTINATION}}) $(kubectl -n {{.TARGET_NAMESPACE}} get pod -l app.kubernetes.io/instance=cli -o jsonpath="{.items[0].metadata.name}"):{{.REMOTE_BACKUP_DESTINATION}}
+        - kubectl exec -n {{.TARGET_NAMESPACE}} deployment/cli -- bash -c "echo Verifying file && test -s {{.REMOTE_BACKUP_DESTINATION}} || (echo {{.REMOTE_BACKUP_DESTINATION}} is missing or empty && exit 1) && echo Dropping database && drush sql-drop -y && echo Importing backup && drush sql:query --file={{.REMOTE_BACKUP_DESTINATION}} && echo Deleting backup && rm {{.REMOTE_BACKUP_DESTINATION}}"
```

Comment:
I would like to split this into multiple lines but I could not get this to work in a combination of Taskfile and `kubectl`. I kept running into syntax error issues.

If you have suggestions then please advice.